.golangci.yml

```diff
@@ -4,7 +4,7 @@ run:
 linters:
   # also see https://golangci-lint.run/usage/linters/#enabled-by-default
   enable:
-    - goimports
+    - gofumpt
```

Comment:
Is `gofumpt` a superset of `goimports`? I know it's a superset of `gofmt`, but `goimports` is also a superset of `gofmt`

Reply:
Good question. I think the answer is actually "no (but apparently don't worry about it?)" https://github.com/mvdan/gofumpt?tab=readme-ov-file#frequently-asked-questions

Reading that link, looks like we can enable both if we want to.

Reply:
Yea, let's enable both